sql_commands.py

```diff
@@ -95,3 +98,108 @@ def post_new_user(user):
     
     return get_user_by_row('*', id_number[0][0])
  
+
+
+
+
+
+######################################################################
+###################### PRESCRIPTIONS TABLE ###########################
+
+"""
+    id INTEGER PRIMARY KEY,
+    user_id INTEGER NOT NULL,
+    prescription_date TEXT NOT NULL,
+    created_date TEXT NOT NULL, 
+    od TEXT NOT NULL,
+    oi TEXT NOT NULL,
+    addition TEXT,
+    notes TEXT,
+    doctor TEXT   
+"""
+
+def post_prescription(data):
+    data = [(data['user_id'], data['prescription_date'], data['created_date'], data['od'], data['oi'], data['addition'], data['notes'], data['doctor'])]
+    query = 'INSERT INTO prescriptions (user_id, prescription_date, created_date, od, oi, addition, notes, doctor) VALUES (?, ?, ?, ?, ?, ?, ?, ?)'
+
+    conn = sqlite3.connect('users.db')
+
+    c = conn.cursor()
+    c.executemany(query, data)
+        
+    conn.commit()
+
+    prescript_id = [n for n in (c.execute('SELECT MAX(id) FROM prescriptions'))] 
+
+    return get_prescription(prescript_id[0][0])
+
+
+def get_prescription(prescript_id):
+    query_command = 'SELECT * FROM prescriptions WHERE id = {}'.format(prescript_id)
+
+    conn = sqlite3.connect('users.db')
+
+    cursor = conn.cursor()
+    query = cursor.execute(query_command)
+    conn.commit()
+    return query.fetchall()
```

Comment:
Following the previous comment, here should be the formatting of the sql result and return a dictionary with the prescription